ludwig/utils/eval_utils.py

```diff
@@ -245,6 +254,11 @@ def stats(self):
             "avg_f1_score_weighted": self.avg_f1_score(average="weighted"),
             "kappa_score": self.kappa_score(),
         }
+        if len(self.idx2label) == 2 and set(self.label2idx.keys()) == {"False", "True"}:
```

Comment:
the labels in idx2label are the labels that are there in the original dataset, there is a mechanism for the user to specify which one is the positive one, so i would not limit this just to {"False", "True"}.
Also the same applies to the calls to the score functions, as in sklearn they don't accept only binary values but also other datatypes.

Reply:
@w4nderlust I agree! The reason I did it this way is because for binary features in particular, we've hardcoded the labels that get passed into the ConfusionMatrix class: https://github.com/ludwig-ai/ludwig/blob/master/ludwig/features/binary_feature.py#L320 

I figured that this more or less guarantees that the predictions will always have these 2 labels - True and False. The binary output feature (in combination with the `fallback_true_label` property in the preprocessing parameters for binary output features) will already preprocess and then post-process it so that we only get these 2 labels back. 

What do you think?